pilot/cmd/pilot-discovery/main.go

```diff
@@ -100,7 +100,7 @@ func init() {
 		"Use a Kubernetes configuration file instead of in-cluster configuration")
 	discoveryCmd.PersistentFlags().StringVar(&serverArgs.Mesh.ConfigFile, "meshConfig", "/etc/istio/config/mesh",
 		fmt.Sprintf("File name for Istio mesh configuration. If not specified, a default mesh will be used."))
-	discoveryCmd.PersistentFlags().StringVar(&serverArgs.NetworksConfigFile, "networksConfig", "/etc/istio/config/networks",
+	discoveryCmd.PersistentFlags().StringVar(&serverArgs.NetworksConfigFile, "networksConfig", "/etc/istio/config-networks/meshNetworks",
```

Comment:
Should make istio config file (mesh, networks, maybe others in future) reside in same directory? It is easy to use and maintain, maybe also merge the two configmaps.

Reply:
Discussed it with @rshriram and because of potentially breaking helm update having this in the same config we chose to have a separate config.

I couldn't find a way to place two config files from configmaps in the same mounted volume unless you use subdir or something similar.

Reply:
Lets keep it separate for a while. But can we apply the file watchers to mesh config as well ? Its a simple thing. We probably just have to have a callback that says mesh config changed.. IT can be a noop for now

Reply:
SGTM